torch/csrc/jit/frontend/ir_emitter.cpp

```diff
@@ -1723,19 +1721,58 @@ struct to_ir {
   // raise a
   //
   // We ignore the expression following raise
-  void emitRaise(const SourceRange& loc) {
-    const std::string exception = "Exception";
-    auto string_input = insertConstant(*graph, exception, loc);
-    graph->insert(prim::RaiseException, {string_input}, {}, loc);
+  void emitRaise(const Raise& raise) {
+    auto sv = emitSugaredExpr(raise.expr(), 1);
```

Comment:
Can we move this all to python_sugared_values ? and put all the conversion of the inputs to strings in there.